### PR TITLE
fix some typos in documentation

### PR DESCRIPTION
**Pull request by svyborov**
## Describe this PR

Find some typos in documentation

## Changes

Fix typos

### CODE REVIEW

**Comment by svyborov** [comment]
@all-contributors please add @svyborov for doc

**Comment by svyborov** [comment]
Hi
I can't figure out why the Create or Update job is failing, can you help?

File: content/300-guides/050-database/100-developing-with-prisma-migrate/160-baselining.mdx [hunk 1]
```diff
@@ -29,7 +29,7 @@ Instead of `migrate deploy`, [`db push`](/concepts/components/prisma-migrate/db-
 
 When you add Prisma Migrate to an existing project, your initial migration contains all the SQL required to recreate the state of the database **before you started using Prisma Migrate**:
 
-![The image shows a database labelled 'Existing database', and a list of existing database features next to it - 24 tables, 13 relationships, 92 fields, 3 indexes. An arrow labelled 'represented by' connects the database features list to a box that represents a migration. The existing databases's features are represented by a single migration.](existing-database.png)
+![The image shows a database labelled 'Existing database', and a list of existing database features next to it - 24 tables, 13 relationships, 92 fields, 3 indexes. An arrow labelled 'represented by' connects the database features list to a box that represents a migration. The existing databases features are represented by a single migration.](existing-database.png)
```
Comment (janpio) [inline content/300-guides/050-database/100-developing-with-prisma-migrate/160-baselining.mdx]: To indicate the posession still:
```suggestion
![The image shows a database labelled 'Existing database', and a list of existing database features next to it - 24 tables, 13 relationships, 92 fields, 3 indexes. An arrow labelled 'represented by' connects the database features list to a box that represents a migration. The existing databases' features are represented by a single migration.](existing-database.png)
```
Maybe?

File: content/300-guides/200-deployment/110-deployment-guides/150-deploying-to-azure-functions.mdx [hunk 2]
```diff
@@ -187,18 +187,18 @@ curl ifconfig.me
 Copy the IP from the output and run the following command, replacing `YOUR_PUBLIC_IP` with the IP address and `UNIQUE_DB_SERVER_NAME` with the name from step 4:
 
 ```terminal copy
-az sql server firewall-rule create --resource-group prisma-azure-example --server UNIQUE_DB_SERVER_NAME --name allow-local-acccess --start-ip-address YOUR_PUBLIC_IP --end-ip-address YOUR_PUBLIC_IP
+az sql server firewall-rule create --resource-group prisma-azure-example --server UNIQUE_DB_SERVER_NAME --name allow-local-access --start-ip-address YOUR_PUBLIC_IP --end-ip-address YOUR_PUBLIC_IP
```
Comment (janpio) [inline content/300-guides/200-deployment/110-deployment-guides/150-deploying-to-azure-functions.mdx:190]: Ohoh, this also made it into https://devblogs.microsoft.com/azure-sql/modern-application-development-with-prisma-graphql-rest-and-azure-sql/
@nikolasburk @ruheni

**Review by janpio — APPROVED** [review]

**Comment by janpio** [comment]
No need to worry about that, currently unfortunately the case for external contributions from forks. We can just ignore it.

Your PR looks good! Left one comment for you, and one for my colleagues...

File: content/300-guides/050-database/100-developing-with-prisma-migrate/160-baselining.mdx (hunk 1 above)
Comment (tanberry) [reply content/300-guides/050-database/100-developing-with-prisma-migrate/160-baselining.mdx]: Yes agree, we still need the apostrophe to indicate possessive case. But are we talking about multiple databases (databases' features) or just one (database's features)?  I think just one, so we can leave it as it was originally.

**Review by tanberry — CHANGES_REQUESTED** [review]
wow, thank you so very much @svyborov !! You are my hero; we really appreciate this rather awkwardly long list of typo fixes. ;)

File: content/300-guides/200-deployment/110-deployment-guides/150-deploying-to-azure-functions.mdx (hunk 2 above)
Comment (nikolasburk) [reply content/300-guides/200-deployment/110-deployment-guides/150-deploying-to-azure-functions.mdx:190]: Alex can you reach out to Davide to get this updated?

Comment (ruheni) [reply content/300-guides/200-deployment/110-deployment-guides/150-deploying-to-azure-functions.mdx:190]: I've reached out to Davide.

Comment (ruheni) [reply content/300-guides/200-deployment/110-deployment-guides/150-deploying-to-azure-functions.mdx:190]: Done!

File: content/300-guides/050-database/100-developing-with-prisma-migrate/160-baselining.mdx (hunk 1 above)
Comment (svyborov) [reply content/300-guides/050-database/100-developing-with-prisma-migrate/160-baselining.mdx]: Thanks for your comments, removed this fix :)

**Review by janpio — APPROVED** [review]

**Comment by keerlu** [comment]
Thanks for this @svyborov, and sorry for the long delay. The updates have now been merged in #4275.

**Pull request closed without merging**